cfgov/v1/views/reports.py

```diff
@@ -151,3 +182,51 @@ def get_queryset(self):
             .order_by("-created_at")
             .prefetch_related("tags")
         )
+
+
+class EnforcementActionsReportView(ReportView):
+    header_icon = "doc-full"
+    title = "Enforcement actions report"
+
+    list_export = [
+        "title",
+        "content",
+        "categories.all",
+        "court",
+        "docket_numbers.all",
+        "initial_filing_date",
+        "statuses.all",
+        "products.all",
```

Comment:
If we use the properties on the model, then I don't think we'll need the `process_` functions above for these fields:

```suggestion
        "docket_number_string",
        "initial_filing_date",
        "status_strings",
        "product_strings",
```